tests/test_irsb.py

```diff
@@ -1,39 +1,42 @@
-import angr
-import pyvex
-import archinfo
+import unittest
 
 import ailment
 
-block_bytes = bytes.fromhex("554889E54883EC40897DCC488975C048C745F89508400048C745F0B6064000488B45C04883C008488B00BEA70840004889C7E883FEFFFF")
-block_addr = 0x4006c6
+try:
+    import angr
+    import archinfo
+except:
+    angr = None
+
+try:
+    import pyvex
+except:
+    pyvex = None
+
+
+class TestIrsb(unittest.TestCase):
+    block_bytes = bytes.fromhex("554889E54883EC40897DCC488975C048C745F89508400048C745F0B6064000488B45C04883C008488B00BEA70840004889C7E883FEFFFF")
+    block_addr = 0x4006c6
+
+    @unittest.skipUnless(pyvex, "pyvex required")
+    def test_convert_from_vex_irsb(self):
+        arch = archinfo.arch_from_id('AMD64')
+        manager = ailment.Manager(arch=arch)
+        irsb = pyvex.IRSB(self.block_bytes, self.block_addr, arch, opt_level=0)
+        ablock = ailment.IRSBConverter.convert(irsb, manager)
```

Comment:
For here and the pcode version, is there a way we can use `ablock`? I could just add an `asset ablock`, but I'm unsure if there is something better and more specific that can be done here, instead of just printing a string as was done previously.